install/resources/mysql_db_tables.php

```diff
@@ -1127,6 +1127,8 @@
   suspensiontime int unsigned NOT NULL default '0',
   suspendsignature tinyint(1) NOT NULL default '0',
   suspendsigtime int unsigned NOT NULL default '0',
+  suspendpm tinyint(1) NOT NULL default '0',
+	pmsuspensiontime int unsigned NOT NULL default '0',
```

Comment:
```suggestion
  pmsuspensiontime int unsigned NOT NULL default '0',
```

Minor change to indentation.